packages/rspack/migrations.json

```diff
@@ -5,6 +5,12 @@
       "version": "20.2.0-beta.3",
       "description": "Update the withModuleFederation import use @nx/module-federation/rspack.",
       "factory": "./src/migrations/update-20-2-0/migrate-with-mf-import-to-new-package"
+    },
+    "ensure-nx-module-federation-package": {
```

Comment:
This is technically already handled by Angular and React plugins right? Is there a scenario where those aren't installed by Rspack is?

Reply:
Each plugin is finding its own set of `module-federation-dev-server` executors to determine if MF projects exists. 

I could combine the rspack ones into the react one, i just had them separated based on location of the executor